### PR TITLE
Fix github action runs on 1.7.8.x

### LINKED ISSUE #32915: The CI for 1.7.8.x is locked
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

All Github Actions are not started

### Expected behavior

Some Github Actions are executed and :green_circle: 

### Steps to reproduce

Create a PR on 1.7.8.x

### PrestaShop version(s) where the bug happened

1.7.8.x

### PHP version(s) where the bug happened

N/A

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

@PrestaShopCorp

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Fix github action runs on 1.7.8.x
| Type?             | bug fix
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | no
| Fixed ticket?     | Partially fixes #32915
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by boherm — DISMISSED** [review]

File: .github/workflows/ui-tests-code-checks.yml [hunk 1]
```diff
@@ -19,6 +19,11 @@ jobs:
       - name: Checkout the repository
         uses: actions/checkout@v2
 
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
```
Comment (leemyongpakvn) [inline .github/workflows/ui-tests-code-checks.yml:24]: I guess we can use Node 16 and actions v3 here.

Comment (boubkerbribri) [reply .github/workflows/ui-tests-code-checks.yml:24]: We can use V3 and node 14 with npm 6 following these recommendations https://devdocs.prestashop-project.org/1.7/development/compile-assets/

Comment (boubkerbribri) [reply .github/workflows/ui-tests-code-checks.yml:24]: done

**Review by matthieu-rolland — COMMENTED** [review]
seems that CI fails at install

**Comment by boubkerbribri** [comment]
> seems that CI fails at install

Yes, this PR will not fix the CI, but it should run.

**Review by boherm — DISMISSED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Pull request merged**